src/CommunityToolkit.Maui.BindablePropertySG/Helpers/HashCode.cs

```diff
@@ -0,0 +1,188 @@
+﻿// https://github.com/CommunityToolkit/dotnet/blob/main/src/CommunityToolkit.Mvvm.SourceGenerators/Helpers/HashCode.cs
```

Comment:
Again I grabbed some code from Sergio, this is used by the `EquatableArray<T>`, and it's necessary because we are on `.netstandard`. And because of this puppy we have to allow `unsafe` code

Yeah, I know... We are limited by the technology of our era